scripts/flink-jar-download.sh

```diff
@@ -4,11 +4,8 @@ set -euo pipefail
 HERE="$( cd "$( dirname "$0" )" && pwd )"
 
 # URL of the file to download
-KAFKA_CONNECTOR_FILE="flink-connector-kafka-3.4.0-1.20.jar"
-KAFKA_CONNECTOR="https://repo.maven.apache.org/maven2/org/apache/flink/flink-connector-kafka/3.4.0-1.20/${KAFKA_CONNECTOR_FILE}"
-
-KAFKA_CLIENTS_FILE="kafka-clients-3.4.0.jar"
-KAFKA_CLIENTS="https://repo1.maven.org/maven2/org/apache/kafka/kafka-clients/3.4.0/${KAFKA_CLIENTS_FILE}"
+KAFKA_CONNECTOR_FILE="flink-sql-connector-kafka-3.4.0-1.20.jar"
+KAFKA_CONNECTOR="https://repo.maven.apache.org/maven2/org/apache/flink/flink-sql-connector-kafka/3.4.0-1.20/${KAFKA_CONNECTOR_FILE}"
```

Comment:
I figured out we do not need both Jars if we use the fat dependency: flink-sql-connector